.circleci/config.yml

```diff
@@ -150,6 +150,27 @@ jobs:
               --pacticipant="Accredited Programmes UI" --version="$CIRCLE_SHA1" --tag="<< parameters.tag >>" \
               --broker-base-url="$PACT_BROKER_BASE_URL" --broker-username="$PACT_BROKER_USERNAME" --broker-password="$PACT_BROKER_PASSWORD"
 
+  end_to_end_test:
+    docker:
+      - image: mcr.microsoft.com/playwright:v1.34.2-focal
+    circleci_ip_ranges: true # opt-in to jobs running on a restricted set of IPs
+    steps:
+      - run:
+          name: Clone E2E repo
+          command: |
+            git clone https://github.com/ministryofjustice/hmpps-accredited-programmes-e2e.git .
+      - run:
```

Comment:
v minor but is installing the latest version of `npm` necessary?

Reply:
So it looks like it came from https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/50/commits/ad97f5abffcfe87ce6b88ef1451a39aff7b3493b, which I'd imagine is copying from Approved Premises, e.g. https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/173/commits/9633dcc4a4a8c6b7a35c0d178988151cf5d3ce99 and https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/670/commits/35fadeaa33611f7b8f4bdf93bc60e6d51f0174d4

I think we might need to update the artifacts path though: https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/904/commits/7aee650ae4d24f97dc4e88587944cddde7180994